README.md

```diff
@@ -7,15 +7,16 @@ This is a Blender add-on that allows importing and exporting 3MF files.
 Installation
 ----
 To install this add-on, currently you need to manually copy the source code into Blender's add-ons folder. Here is how you do that:
-1. Download the source code from the [latest release](https://github.com/Ghostkeeper/Blender3mfFormat/releases/latest). You can find it under "Assets".
-2. Extract the `io_mesh_3mf` folder in the zip archive to Blender's add-ons folder. The location of that folder varies depending on your operating system:
+1. Download the source code from the [latest release](https://github.com/Ghostkeeper/Blender3mfFormat/releases/latest). You can find it under "Assets", or use the Clone or Download button for the latest source version.
+2. Extract the `Blender3mfFormat-0.1.0` folder (version 0.1.0) or `io_mesh_3mf` folder (version 0.2.0 and later) in the zip archive to Blender's add-ons folder. The location of that folder varies depending on your operating system:
```

Comment:
I think it's not a good idea to document the installation steps for all historic versions here. It'll muddle the instructions with unnecessary information for most people. If they really need to learn how to install an older version, they'd have to review the README.md file that came with the version they downloaded.

However there was no README.md file with the 0.1 release. That's on me, but in spirit this still holds.

Reply:
Sounds good, do you want me to make those tweaks and resubmit? The main reason I had them in there was because the version in the Assets that it mentions to download is 0.1.0 so had a different folder name.

Reply:
Pull request etiquette says I should allow you to make the change first :stuck_out_tongue: So go ahead. Otherwise I'll fix it up.

You don't need to resubmit. You can make an additional commit on the readme-md-update branch in your repository and it'll update here.

The readme doesn't mention 0.1.0. Just the latest release, which is 0.2.0.

Reply:
Thanks for that... I am a bit of a newbie at pull requests etiquette... Made the change :-).

In terms of where I got the other folder from in the first place, that was by following the original instructions. So 0.1.0 isn't mentioned in the readme, but by following the instructions and clicking on the [latest release] link, then Assets, that is how I landed up with the older version :-)

Reply:
Ah, well that page lists both versions. Each has their own assets. The confusion is understandable.